src/test/keyboard-shortcut.service.spec.ts

```diff
@@ -219,19 +219,32 @@ describe('KeyboardShortcutService', () => {
                     handler: () => (higherPriorityResult = true),
                     description: '',
                     priority: 3,
-                    passToLowerPriorities: true
+                    passToLowerPriorities: true,
                 });
                 service.listen({
                     keyBinding: keyCombination,
                     handler: () => (lowerPriorityResult = true),
                     description: '',
-                    priority: 3
+                    priority: 3,
                 });
                 const event = <KeyboardEvent>{ altKey: true, key: 'C' };
                 service.sendKeyboardEventToHandler(event);
                 expect(higherPriorityResult).not.toBeNull();
                 expect(lowerPriorityResult).not.toBeNull();
             });
         });
+
+        describe(':: testing handleKeyboardEvent', () => {
+            it('should trigger the handler when key event matches', () => {
+                const event = <KeyboardEvent>{ altKey: true, key: '' };
+                service.sendKeyboardEventToHandler(event);
```

Comment:
these are not actually doing any expects.  That's kind of a no-no since this would be considered cheating the code coverage.  You are causing the statements to run without checking anything meaningful occurred.